mivs/templates/mivs_applications/index.html

```diff
@@ -72,7 +72,7 @@
 {% endif %}
 
 {% if c.ALLOWED_TO_SUBMIT_ROUND1 %}
-    {% if c.HAS_INDIE_ADMIN_ACCESS %}
+    {% if c.HAS_INDIE_ADMIN_ACCESS and c.AFTER_ROUND_ONE_DEADLINE %}
```

Comment:
This and the similar change below is unrelated, but I noticed that this message would appear claiming that only admins can create a studio or game, but that wasn't always the case.